Nos/Views/HomeFeedView.swift

```diff
@@ -60,55 +63,78 @@ struct HomeFeedView: View {
     }
 
     var body: some View {
```

Comment:
This just adds the top horizontal scrollbar to navigate through avatars of authors with stories